doc/escapable_handle_scope.md

```diff
@@ -40,16 +40,16 @@ Napi::EscapableHandleScope Napi::EscapableHandleScope::New(napi_env env, napi_ha
 
 Returns a new `Napi::EscapableHandleScope` instance which wraps the
 napi_escapable_handle_scope handle passed in. This can be used
-to mix usage of the C N-API and node-addon-api.
+to mix usage of the C Node-API and node-addon-api.
 
 operator EscapableHandleScope::napi_escapable_handle_scope
 
 ```cpp
 operator Napi::EscapableHandleScope::napi_escapable_handle_scope() const
 ```
 
-Returns the N-API napi_escapable_handle_scope wrapped by the `Napi::EscapableHandleScope` object.
-This can be used to mix usage of the C N-API and node-addon-api by allowing
+Returns the Node-API napi_escapable_handle_scope wrapped by the `Napi::EscapableHandleScope` object.
```

Comment:
Could you use backtick for napi_escapable_handle_scope? From napi_escapable_handle_scope to `napi_escapable_handle_scope`